### PR TITLE
Feature/page design

### PR DESCRIPTION
**Pull request by ttraenkler**
I have refined the design of the navigation bar and added mobile support. I replaced the links on the front page with the navigation bar. The navigation items I would see fit are "News, Speakers, Schedule, Venue, About". The about page contains the FAQ, including the contact email, Twitter, and code of conduct links. I have also added refined the text and added some items to the FAQ (e.g. hotel recommendations and the Discord channel) - see the screenshots below. Let me know what you think, this could be merged before continuing on the speaker and schedule page, which in the first instance I would derived from the about page template.

This PR addresses part of #37 - in particular it proposes a new design of the about page and the nav bar which will could be used across pages:

- new design of the FAQ page and nav bar
- adds mobile support (burger menu) for the navbar
- proposes some text changes to the FAQ
- replaces the links on the front page with the nav bar

Bildschirmfoto 2020-01-07 um 03 17 06
Bildschirmfoto 2020-01-07 um 03 18 18
Bildschirmfoto 2020-01-07 um 03 17 09
Bildschirmfoto 2020-01-07 um 03 17 15

### CODE REVIEW

**Comment by ttraenkler** [comment]
Moved the navbar on the front page down as it looks cleaner
Bildschirmfoto 2020-01-07 um 02 51 47

**Comment by ttraenkler** [comment]
same for content pages
Bildschirmfoto 2020-01-07 um 03 48 18

**Comment by ttraenkler** [comment]
Bildschirmfoto 2020-01-07 um 07 18 25

File: components/NavBar/index.tsx [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+const ActiveButton = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin: 0;
+  padding: 0 3%;
+  height: 60px;
+  transition: 150ms;
+  border-top: 2px solid white;
+
```
Comment (torch2424) [inline components/NavBar/index.tsx]: Can we add `cursor: pointer` here? :smile: On my Machine, the cursor becomes the text selection thing. When I think it should be the hand link button looking thing (pointer) :joy:

Comment (ttraenkler) [reply components/NavBar/index.tsx]: Good catch. It seems to be an issue with next.js `<Link>` and styled-components. https://github.com/zeit/next.js/issues/1942 they suggest passing `passHref`. Further, I noticed the padding of the buttons is jumping back and forth if you press them repeatedly, I will review the CSS for conflicting definitions.

**Comment by ashleygwilliams** [comment]
i think my only concern is that the speakers page looks very different than the rest of the site- no purple at all! i think we might want to keep the purple consistent? curious what others think

File: components/NavBar/index.tsx (hunk 1 above)
Comment (ttraenkler) [reply components/NavBar/index.tsx]: I've fixed the link ux issue.

**Comment by ttraenkler** [comment]
@torch2424 I have fixed the link issue and moved the navbar back to top. Please review again

**Comment by ttraenkler** [comment]
> i think my only concern is that the speakers page looks very different than the rest of the site- no purple at all! i think we might want to keep the purple consistent? curious what others think. 

The speaker page is not part of this PR, I've accidentally put the speaker page here, so this concern can be dealt with separately

**Comment by ttraenkler** [comment]
I've attached up to date screenshots of the content of this PR.

Bildschirmfoto 2020-01-07 um 19 08 07
Bildschirmfoto 2020-01-07 um 19 08 15

File: components/NavBar/index.tsx [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+const NavBar: FC<{
+  title?: string;
+  currentPage?: string;
+  bottom?: boolean;
+}> = ({ title, currentPage, bottom }) => {
+  const [open, setOpen] = useState(false);
+  const Box = bottom ? BottomBox : TopBox;
+
+  return (
+    <>
```
[the remark this answers is not in the document]
Comment (ashleygwilliams) [reply components/NavBar/index.tsx]: lol same

**Pull request merged**